packages/renderer/src/lib/ui/ButtonSpec.svelte

```diff
@@ -0,0 +1,6 @@
+<script lang="ts">
+import { faLightbulb } from '@fortawesome/free-solid-svg-icons';
+import Button from './Button.svelte';
+</script>
+
+<Button title="Tooltip" icon="{faLightbulb}">Click me</Button>
```

Comment:
I'm wondering why we have this component ?

couldn't we use icon property in the renderer and only in Button.spec.ts ?

Reply:
I suppose you only need this component for the slot (name to be displayed in the button) but for the icon we don't

Reply:
Yes, that's originally why it's there. It should be possible to move the icon to regular test now, I just didn't bother since we'd still need this unless there's a way to test the default slot, and it felt slightly less mock-y to me to have a component use icon={}? Could move it though, would be nicer if we could merge entirely.